### PR TITLE
NEWS.md: prep for 0.15.0 tag

### LINKED ISSUE #806: update NEWS.md for flux-sched v0.15
We need a tag of flux-sched v0.15 today to go along with flux-core v0.23.

This issue is to track the need for a `NEWS.md` update for the flux-sched release.

### PR DESCRIPTION
**Pull request by SteVwonder**
Closes #806

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+ * Implement unpack_at for JGF (#775)
+ * qmanager: support re-prioritization of jobs (#803)
+ * fluxion-R: add flux-ion-R.py as a public command (#800)
+ * ci: migrate to github actions (#794)
+
+### Fixes
+
+ * qmanager: update per libschedutil API changes (#801)
+ * mergify: fix referenced test name (#804)
+
```
Comment (milroy) [inline NEWS.md:15]:
```suggestion
* traverser: fix segfault for bi-edge traversal (#775)
```

**Review by milroy — COMMENTED** [review]
I made a suggestion to list the segfault fix (part of PR #775) in the "Fixes" category.  Otherwise LGTM!

File: NEWS.md (hunk 1 above)
Comment (SteVwonder) [reply NEWS.md:16]: Thanks @milroy! Sorry for missing that one (I'll admit I just relied on the script and didn't spot check it).  Just pushed with that added.

Comment (SteVwonder) [reply NEWS.md:16]: If that new commit looks good, can you "Approve" and set the MWP label?  I'll finish up the release process and tag tonight if so.

**Review by milroy — APPROVED** [review]
Tnanks, @SteVwonder!  I'm approving this PR and setting MWP.

**Comment by dongahn** [comment]
Thanks folks. LGTM as well!

**Review by dongahn — APPROVED** [review]

**Pull request merged**